db/migrate/20151103223107_drop_properties.rb

```diff
@@ -0,0 +1,5 @@
+class DropProperties < ActiveRecord::Migration
+  def change
```

Comment:
should we mark this migration is irreverisble? http://jacopretorius.net/2014/07/creating-irreversible-migrations-in-rails.html